components/ReplyItem.js

```diff
@@ -132,12 +133,14 @@ ReplyItem.fragments = {
       text
       type
       createdAt
-      user {
-        id
-        name
-        level
-        avatarUrl
-      }
+    }
```

Comment:
We may need to add `ReplyInfo.fragments.replyInfo` in `ReplyItem.fragments.ReplyItem`, since `ReplyInfo` actually uses `ReplyInfo`.